lib/challenge.js

```diff
@@ -19,8 +19,34 @@
   // - park
 
 // Answer Starts Here
-
-// Answer Ends Here
+// 'Calendar'
+//Description: Our calendar keeps track of things.
+// Every 'Calendar' has
+// - the months of the year
+// - days of the week
+// - weekly to-do list
+//Every 'Calendar' can
+// - navigate between dates
+// - add items to the to-do list
+// - send reminders
+//
+Every 'Calendar' has
+- the months of year
+days of the week
+-national holidays
+Every 'Calendar' can
+- schedule meetings
+- repeat frequent appointments
+- toggle between dates
+
+Every 'Calendar' has
+-days of the week
+- a daily to-do list
+- a monthly to-do list
+Every 'Calendar' can
+-toggle between dates
+-generate to-do lists
+-send reminders// Answer Ends Here
```

Comment:
Great. The question asks for three such big-picture models, and you've got three.

One thing to note is the answer to this question is plain English, and this is a file containing code. Plain English belongs strictly in comments so that it doesn't prevent the file from being run as code. As a result, running the tests always results in a complete failure no matter how correct your answers are -- there are syntax errors at the very top of the file.

1